htsget/htsget_test.go

```diff
@@ -58,8 +58,8 @@ access_token = eyJ0eXAiOiJKV1QiLCJqa3UiOiJodHRwczovL29pZGM6ODA4MC9qd2siLCJhbGciO
 	if err != nil {
 		panic(err)
 	}
-	os.Args = []string{"htsget", "-config", tmpDir + "s3cmd_test.conf", "-dataset", "DATASET0001", "-filename", "htsnexus_test_NA12878", "-host", "somehost", "-pubkey", "somekey"}
-	err = Htsget(os.Args)
+	os.Args = []string{"htsget", "-dataset", "DATASET0001", "-filename", "htsnexus_test_NA12878", "-host", "somehost", "-pubkey", "somekey"}
+	err = Htsget(os.Args, tmpDir+"s3cmd_test.conf")
```

Comment:
Although it is the test code, I think it is better to use a library to concatenate path instead of just using `+`.
E.g.

```golang
err = Htsget(os.Args, filepath.Join(tmpDir, "s3cmd_test.conf"))
```
So as for the other places